plugins/inputs/wireless/wireless_test.go

```diff
@@ -43,11 +45,15 @@ func TestLoadWirelessTable(t *testing.T) {
 			Beacon:    int64(0),
 		},
 	}
-	metrics, err := loadWirelessTable(testInput)
+
+	w := Wireless{
+		Log: testutil.Logger{},
+	}
+	metrics, err := w.loadWirelessTable(testInput)
```

Comment:
The follow-up lines could be replaced by `require.NoError()`...

Reply:
Done